irohad/ametsuchi/impl/postgres_command_executor.cpp

```diff
@@ -0,0 +1,770 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "ametsuchi/impl/postgres_command_executor.hpp"
+
+#include <boost/format.hpp>
+
+#include "ametsuchi/impl/postgres_wsv_common.hpp"
+
+namespace iroha {
+  namespace ametsuchi {
+
+    expected::Error<CommandError> makeCommandError(
```

Comment:
Please put all local functions in an anonymous namespace. This will make them link internally and not pollute ametsuchi namespace.